src/program.tsx

```diff
@@ -49,7 +49,7 @@ class Program extends React.Component<ProgramProps, ProgramState> {
     console.log("[Gongfu] Running Update for ", msg.tag);
 
     const { update } = this.props;
-    const { model, effect } = update(msg, this.state.model);
+    const [model, effect] = update(msg, this.state.model);
```

Comment:
Yeah exactly! I feel like this will improve all the boilerplate involved a little